test/karma/behaviorSpec.js

```diff
@@ -0,0 +1,522 @@
+var _ = require('underscore');
+var TorsoBehavior = require('./../../modules/Behavior');
+var TorsoView = require('./../../modules/View');
+
+var spyOnBackbone = require('./helpers/spyOnBackbone');
```

Comment:
You can't use spyOnBackbone as currently implemented to track things that happen during initialize, because it wraps the "initialize" method.